### PR TITLE
Add sample/README_mysql.md

### PR DESCRIPTION
**Pull request by revit13**

### CODE REVIEW

File: sample/README_mysql.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+# Mysql Sample
+
+This sample is for experimenting with the Airbyte Module (ABM) mysql connetor.
+
+The server is configured to read and write a mysql table.
+
+It uses Kuberentes for the mysql chart deployment only while the Airbyte Module (ABM) runs locally without Kubernetes.
```
Comment (cdoron) [inline sample/README_mysql.md:7]: I suggest: `It uses Kuberentes only for the mysql chart deployment, while the Airbyte Module (ABM) runs locally without Kubernetes.`

File: sample/README_mysql.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      kubectl config set-context --current --namespace=fybrik-airbyte-sample
+      ```
+
+    2. Deploy [mysql](https://bitnami.com/stack/mysql/helm) helm chart in `fybrik-airbyte-sample` namespace.
+      ```bash
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      kubectl wait pod --for=condition=ready mysql-0 --namespace fybrik-airbyte-sample --timeout 20m
+      ```
+    3. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
```
Comment (cdoron) [inline sample/README_mysql.md:38]: "run a pod that is **used** as a client"

File: sample/README_mysql.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      (1,'PAYMENT','1864.28','C1666544295','21249','19384.72','M2044282225','0','0',0,0),
+      (1,'TRANSFER','181','C1305486145','181','0','C553264065','0','0',1,0),
+      (1,'CASH_OUT','181','C840083671','181','0','C38997010','21182','0',1,0),
+      (1,'PAYMENT','11668.14','C2048537720','41554','29885.86','M1230701703','0','0',0,0),
+      (1,'PAYMENT','7817.71','C90045638','53860','46042.29','M573487274','0','0',0,0);
+      ```
+
+    5. exit the `mysql` prompt and `mysql-client` pod
+
+    6. port foward mysql service:
```
Comment (cdoron) [inline sample/README_mysql.md:61]: foward -> forward

File: sample/README_mysql.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      kubectl port-forward svc/mysql -n fybrik-airbyte-sample 3306:3306 &
+      ```
+
+1. Install python dependencies
+    ```bash
+    pipenv install
+    ```
+1. Update `password` field in the configuration
+   ```bash
+   yq  e -i '.data[0].connection.read_mysql.password=env(MYSQL_ROOT_PASSWORD)' sample/read_mysql.yaml
```
Comment (cdoron) [inline sample/README_mysql.md:72]: cool!

File: sample/README_mysql.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+      kubectl config set-context --current --namespace=fybrik-airbyte-sample
+      ```
+
+    2. Deploy [mysql](https://bitnami.com/stack/mysql/helm) helm chart in `fybrik-airbyte-sample` namespace:
+      ```bash
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      kubectl wait pod --for=condition=ready mysql-0 --namespace fybrik-airbyte-sample --timeout 20m
+      ```
+    3. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
```
Comment (cdoron) [inline sample/README_mysql.md:99]: "use" -> "used"

File: sample/README_mysql.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      ```
+   
+    4. In a mysql client shell prompt insert the following commands to create a new database called `test`:
+      ```bash
+      create database test;
+      ```
+
+    5. exit the `mysql` prompt and `mysql-client` pod
+
+    6. port foward mysql service:
```
Comment (cdoron) [inline sample/README_mysql.md:114]: foward -> forward

File: sample/README_mysql.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
+# Mysql Sample
+
+This sample is for experimenting with the Airbyte Module (ABM) mysql connetor.
```
[the remark this answers is not in the document]
Comment (revit13) [reply sample/README_mysql.md]: Fixed. Thanks

File: sample/README_mysql.md (hunk 2 above)
Comment (revit13) [reply sample/README_mysql.md]: Fixed. Thanks

File: sample/README_mysql.md (hunk 1 above)
Comment (revit13) [reply sample/README_mysql.md]: Fixed. Thanks

File: sample/README_mysql.md (hunk 5 above)
Comment (revit13) [reply sample/README_mysql.md]: Fixed. Thanks

File: sample/README_mysql.md (hunk 6 above)
Comment (revit13) [reply sample/README_mysql.md]: Fixed. Thanks

**Review by cdoron — APPROVED** [review]

**Pull request merged**